### PR TITLE
Feature/alex map attribute to tf

### PR DESCRIPTION
**Pull request by alexazarh**

### CODE REVIEW

File: package/cloudshell/iac/terraform/services/svc_attribute_handler.py [hunk 1]
```diff
@@ -12,3 +12,13 @@ def get_attribute(self, attribute_name: str) -> str:
         if f"{self._tf_service.cloudshell_model_name}.{attribute_name}" in self._attributes:
             return self._attributes[f"{self._tf_service.cloudshell_model_name}.{attribute_name}"]
         return ""
+
+    def check_attribute_exist(self, attribute_name: str) -> bool:
+        return attribute_name in self._tf_service.attributes or \
+               f"{self._tf_service.cloudshell_model_name}.{attribute_name}" in self._tf_service.attributes
```
Comment (qualidan) [inline package/cloudshell/iac/terraform/services/svc_attribute_handler.py]: why not -
        return attribute_name in self._tf_service.attributes or check_2nd_gen_attribute_exist(attribute_name)

File: package/cloudshell/iac/terraform/services/svc_attribute_handler.py [hunk 2]
```diff
@@ -12,3 +12,13 @@ def get_attribute(self, attribute_name: str) -> str:
...
+
+    def check_2nd_gen_attribute_exist(self, attribute_name: str) -> bool:
+        return f"{self._tf_service.cloudshell_model_name}.{attribute_name}" in self._tf_service.attributes
```
Comment (qualidan) [inline package/cloudshell/iac/terraform/services/svc_attribute_handler.py]: why not - 
return (get_2nd_gen_attribute_full_name(attribute_name) in self._tf_service.attributes)

File: package/cloudshell/iac/terraform/models/config.py [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+
+
 class TerraformShellConfig:
-    def __init__(self, write_sandbox_messages: bool = False, update_live_status: bool = False):
+    def __init__(self, write_sandbox_messages: bool = False, update_live_status: bool = False,
+                 inputs_map: Dict = None, outputs_map: Dict = None):
         self.write_sandbox_messages = write_sandbox_messages
         self.update_live_status = update_live_status
+        self.inputs_map = inputs_map
+        self.outputs_map = outputs_map
```
Comment (qualidan) [inline package/cloudshell/iac/terraform/models/config.py:11]: Just wondering why not hold a private InputOutputService isntead of these two variables

File: package/cloudshell/iac/terraform/services/input_output_service.py [hunk 4]
```diff
@@ -10,63 +11,89 @@
...
-            # remove the prefix to get the TF variable name
-            value = self._driver_helper.tf_service.attributes[var]
-            value = self.try_decrypt_password(value)
-            tf_var = var.replace(var_prefix, "")
+        # find all attributes that end with "_tfvar"
+        for attribute_name in self._driver_helper.tf_service.attributes:
+            # add tf variable specific attributes to result
+            if m := re.match(self._var_postfix_regex, attribute_name):
+                # remove the prefix to get the TF variable name
+                value = self._driver_helper.tf_service.attributes[attribute_name]
```
Comment (qualidan) [inline package/cloudshell/iac/terraform/services/input_output_service.py]: Why not use the ServiceAttrHandler ?

File: package/cloudshell/iac/terraform/services/svc_attribute_handler.py (hunk 2 above)
Comment (alexazarh) [reply package/cloudshell/iac/terraform/services/svc_attribute_handler.py]: correct

File: package/cloudshell/iac/terraform/services/svc_attribute_handler.py (hunk 1 above)
Comment (alexazarh) [reply package/cloudshell/iac/terraform/services/svc_attribute_handler.py]: correct

File: package/cloudshell/iac/terraform/models/config.py (hunk 3 above)
Comment (alexazarh) [reply package/cloudshell/iac/terraform/models/config.py:11]: wont fix

File: package/cloudshell/iac/terraform/services/input_output_service.py (hunk 4 above)
Comment (alexazarh) [reply package/cloudshell/iac/terraform/services/input_output_service.py]: will fix

**Pull request merged**